### ISSUE TITLE
A11y_Manage.IIS.Net_Are you sure Popup_Screen Reader: Narrator is not reading the message in the popup

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/IIS.WebManager#212]
**Environment:**
https://manage.iis.net/connect 
Browser: Microsoft Edge 42.17134.1.0, Google Chrome 69.0.3497.100

**MAS ID Violated:**
MAS 1.3.1

**Repro Steps:**
1. Hit the URL "https://manage.iis.net" 
2. Tab till "Download Microsoft IIS Administration" button and press enter and run "IISAdministrationSetup.exe" file.
3. Check the check box and tab till "Install" button and press enter.
4. Tab till "Cancel" button and press enter.
5. Verify narrator is reading the message in the popup or not

**Actual Result:**
Narrator is not reading the message in the popup

**Expected Result:**
Narrator should read the message in the popup

**Note:**
Similar issue is observed in the following pages:
1.Setup Progress Popup
2.User cancelled Popup
3.Setup Successful Popup

**User Impact:**
Visually impaired users finds difficult to understand if narrator does not read the message in the popup

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug_212 _Narrator not reading message in the popup.pptx](https://github.com/Microsoft/IIS.WebManager/files/2433904/Bug_212._Narrator.not.reading.message.in.the.popup.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This UI is not used anymore.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Edge; #Narrator

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.iis.net%2F&data=05%7C01%7Cv-bhaprakash%40microsoft.com%7C7a737faa4af14c4ad31a08db634ab100%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638212945750407456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Q7lvppcGrDLIRxk6EaLiumHk2i6E5S784T%2BniobLEu4%3D&reserved=0) home page will be removed.
